src/test/java/com/buschmais/jqassistant/plugin/rdbms/scanner/SchemaScannerPluginIT.java

```diff
@@ -59,12 +59,18 @@ void createStructures() throws SQLException, ClassNotFoundException {
             execute(c, "drop view if exists PERSON_VIEW");
             execute(c, "drop table if exists PERSON");
             execute(c, "create table PERSON(a decimal(10,5), b decimal(5,2), c varchar(255) default 'defaultValue')");
+            execute(c, "comment on table PERSON is 'table remarks'");
+            execute(c, "comment on column PERSON.a is 'column remarks'");
             execute(c, "alter table PERSON add constraint PK_PERSON primary key (A,B)");
             execute(c, "create table ADDRESS(PERSON_A decimal(10,5), PERSON_B decimal(5,2))");
             execute(c, "alter table ADDRESS add constraint FK_ADDRESS_PERSON foreign key (PERSON_A,PERSON_B) references PERSON(A,B)");
             execute(c, "create sequence PERSON_SEQ minvalue 100 maxvalue 10000  start with 100 increment by 10 cycle");
+            // FIXME: hsqldb should support commenting on sequence, but it fails. needs more investigation
+//            execute(c, "comment on SEQUENCE PERSON_SEQ is 'sequence remarks'");
             execute(c, "create view PERSON_VIEW as select a from PERSON");
             execute(c, "create trigger PERSON_TRIGGER after insert ON PERSON when (true) delete from PERSON");
+            // FIXME: hsqldb should support commenting on sequence, but it fails. needs more investigation
```

Comment:
A pull request shouldn't add technical debts like this. If does not support comments on triggers, we could exlude it from the tests.